cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -120,9 +68,9 @@ func buildDaisyVars(source resource, translateWorkflowPath, imageName, family, d
 		varMap["sysprep_windows"] = strconv.FormatBool(sysprepWindows)
 	}
 	if isFile(source) {
-		varMap["source_disk_file"] = source.path()
+		varMap["source_disk_file"] = source.Path()
 	} else {
-		varMap["source_image"] = source.path()
+		varMap["source_image"] = source.Path()
 	}
 	varMap["family"] = strings.TrimSpace(family)
 	varMap["description"] = strings.TrimSpace(description)
```

Comment:
is it double trimmed?

Reply:
It is -- there are a bunch of tests that assert that these are trimmed. They'll be going away in the next chunk.